modules/gdscript/tests/scripts/runtime/errors/constant_array_is_deep.out

```diff
@@ -0,0 +1,6 @@
+GDTEST_RUNTIME_ERROR
+>> SCRIPT ERROR
+>> on function: test()
+>> runtime/errors/constant_array_is_deep.gd
+>> 6
+>> Invalid set index '0' (on base: 'Dictionary') with value of type 'int'
```

Comment:
Note for future work: we should make this error message more explicit.

Reply:
Also, hopefully as an analyzer error rather than as a runtime error?